packages/web/package.json

```diff
@@ -35,8 +35,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "lint": "eslint --fix src --ext .ts,.tsx"
```

Comment:
I didn't mean to remove the full line, only the `--fix` option. You can keep the link script.